config/config.default.yaml

```diff
@@ -923,20 +923,10 @@ costs:
     lifetime: 25
     "CO2 intensity": 0
     "discount rate": 0.07
+  custom_costs:
+    enable: false
+    file: data/custom_costs.csv
   overwrites: {}
-  marginal_cost:
-    solar: 0.01
-    onwind: 0.015
-    offwind: 0.015
-    hydro: 0.
-    H2: 0.
-    electrolysis: 0.
-    fuel cell: 0.
-    battery: 0.
-    battery inverter: 0.
-    home battery storage: 0
-    water tank charger: 0.03
-    central water pit charger: 0.025
```

Comment:
Have these values been previously applied in the default configuration? If yes, we should also keep them as defaults (maybe move them into the `data/custom_costs.csv`?